Likage error in android buiding

commit 74e996185d4d69a9469a6717342fc02839251e19
Host system Linux x86_64

I am getting this error after lasts commit in ogre3d in android compilation

100%] Linking CXX shared library ../../SampleBrowserNDK/app/src/main/jniLibs/armeabi-v7a/libOgreSampleBrowser.so
src/EGL/OgreEGLSupport.cpp:77: error: undefined reference to 'eglGetPlatformDisplay'
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [Samples/Browser/CMakeFiles/SampleBrowser.dir/build.make:137: SampleBrowserNDK/app/src/main/jniLibs/armeabi-v7a/libOgreSampleBrowser.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:1018: Samples/Browser/CMakeFiles/SampleBrowser.dir/all] Error 2
make: *** [Makefile:156: all] Error 2

--- a/RenderSystems/GLSupport/src/EGL/OgreEGLSupport.cpp
+++ b/RenderSystems/GLSupport/src/EGL/OgreEGLSupport.cpp
@@ -58,7 +58,7 @@ namespace Ogre {
 
     EGLDisplay EGLSupport::getGLDisplay(void)
     {
-#ifdef EGL_VERSION_1_5
+#if defined(EGL_VERSION_1_5) && OGRE_PLATFORM != OGRE_PLATFORM_ANDROID
         static auto eglQueryDevicesEXT = (PFNEGLQUERYDEVICESEXTPROC)eglGetProcAddress("eglQueryDevicesEXT");
         static auto eglQueryDeviceStringEXT =
             (PFNEGLQUERYDEVICESTRINGEXTPROC)eglGetProcAddress("eglQueryDeviceStringEXT");

